.github/workflows/ci.yml

```diff
@@ -12,21 +12,17 @@ jobs:
     uses: ./.github/workflows/cancel_builds.yml
   validate_imported_protos:
     uses: ./.github/workflows/validate_imported_protos.yml
-  validate_mi_mutation:
```

Comment:
Note: Talked with Dan and this validation step is busted. Going to create some tech debt to see if we can get a viable script like we have for validating imported protos vs. generated but as of right now its not going to work so we're just disabling it on the CI.